src/org/usfirst/frc/team694/robot/subsystems/Lift.java

```diff
@@ -17,51 +19,69 @@
  */
 public class Lift extends Subsystem {
 
-    private WPI_TalonSRX leftLiftMotor;
-    private WPI_TalonSRX rightLiftMotor; 
+    private WPI_TalonSRX innerLeftLiftMotor;
+    private WPI_TalonSRX innerRightLiftMotor;
+    private WPI_VictorSPX outerLeftLiftMotor;
+    private WPI_VictorSPX outerRightLiftMotor;
     
     private Solenoid liftSolenoid; 
 
     private boolean brakeOn; 
   
     public Lift() {
+        innerLeftLiftMotor = new WPI_TalonSRX(RobotMap.INNER_LEFT_LIFT_MOTOR_PORT);
+        innerRightLiftMotor = new WPI_TalonSRX(RobotMap.INNER_RIGHT_LIFT_MOTOR_PORT);
+        
+        outerLeftLiftMotor = new WPI_VictorSPX(RobotMap.OUTER_LEFT_LIFT_MOTOR_PORT);
+        outerRightLiftMotor = new WPI_VictorSPX(RobotMap.OUTER_RIGHT_LIFT_MOTOR_PORT);
 
-        leftLiftMotor = new WPI_TalonSRX(RobotMap.LEFT_LIFT_MOTOR_PORT);
-        rightLiftMotor = new WPI_TalonSRX(RobotMap.RIGHT_LIFT_MOTOR_PORT);
-
-        leftLiftMotor.setNeutralMode(NeutralMode.Brake);
-        rightLiftMotor.setNeutralMode(NeutralMode.Brake);
+        innerLeftLiftMotor.setNeutralMode(NeutralMode.Brake);
+        innerRightLiftMotor.setNeutralMode(NeutralMode.Brake);
+        
+        outerLeftLiftMotor.setNeutralMode(NeutralMode.Brake);
+        outerRightLiftMotor.setNeutralMode(NeutralMode.Brake);
 
-        rightLiftMotor.setInverted(true);
+        innerRightLiftMotor.setInverted(true);
+        outerRightLiftMotor.setInverted(true);
 
-        rightLiftMotor.follow(leftLiftMotor);
+        innerRightLiftMotor.follow(innerLeftLiftMotor);
+        outerRightLiftMotor.follow(innerLeftLiftMotor);
+        outerLeftLiftMotor.follow(innerLeftLiftMotor);
   
-        leftLiftMotor.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder, 0, 0);
-        rightLiftMotor.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder, 0, 0);
+        innerLeftLiftMotor.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder, 0, 0);
+        innerRightLiftMotor.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder, 0, 0);
 
         liftSolenoid = new Solenoid(RobotMap.LIFT_BRAKE_SOLENOID_CHANNEL);
         
         // Configures the limit switches (forward is top, reverse is bottom)
-        leftLiftMotor.configForwardLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen, 10);
-        leftLiftMotor.configReverseLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen, 10);
+        innerLeftLiftMotor.configForwardLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen, 10);
+        innerLeftLiftMotor.configReverseLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen, 10);
```

Comment:
Confirm with engineering on whether we have a limit switch at the top of the lift (last I heard, we only have one limit switch at the bottom of the lift)

Reply:
We will have two limit switches, confirmed with Mr. Blay and Eric.